extensions/amp-video-docking/0.1/def.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @enum {string} */
+export const RelativeX = {LEFT: 'left', RIGHT: 'right'};
+
+/** @enum {string} */
+export const RelativeY = {TOP: 'top', BOTTOM: 'bottom'};
```

Comment:
Looks like this is only used for scroll direction anymore so can we rename it to `ScrollDirection`?

Reply:
`DirectionX` and `DirectionY` as compromise ;)